src/partr.c

```diff
@@ -260,12 +359,12 @@ static jl_task_t *get_next_task(jl_value_t *getsticky)
     return multiq_deletemin();
 }
 
+extern volatile unsigned _threadedregion;
```

Comment:
Can we have a comment what this implies now and have the name reflect that?

-- edit saw the later comment.